### PR TITLE
🔖(minor) prepare 0.2.0 distribution release

### PR DESCRIPTION
**Pull request by wilbrdt**
### Changed

- Upgrade base Warren images to 0.2.0
- Rework frontend to connect to now stable API

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,5 +20,6 @@ and this project adheres to
 - Add a custom API Docker image, based on `warren:api-core-0.1.0`, for the tdbp plugin indicator
 - Add a custom APP Docker image, based on `warren:app-0.1.0`, to serve the Warren-TdBP frontend application
 
-[unreleased]: https://github.com/apui-avignon-university/warren-tdbp/compare/v0.1.0...main
+[unreleased]: https://github.com/apui-avignon-university/warren-tdbp/compare/v0.2.0...main
+[0.1.0]: https://github.com/apui-avignon-university/warren-tdbp/compare/v0.1.0...v0.2.0
```
Comment (quitterie-lcs) [inline CHANGELOG.md]:
```suggestion
[0.2.0]: https://github.com/apui-avignon-university/warren-tdbp/compare/v0.1.0...v0.2.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.2.0] - 2024-05-23
```
Comment (quitterie-lcs) [inline CHANGELOG.md:11]:
```suggestion
## [Unreleased]

## [0.2.0] - 2024-05-23
```

File: CHANGELOG.md (hunk 1 above)
Comment (wilbrdt) [reply CHANGELOG.md]: Nice catch, thanks!

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**